rspace/src/main/tut/Tutorial.md

```diff
@@ -408,9 +408,69 @@ val pres14 = space.produce(Channel("friends"), erin, persist = true)
 println(space.store.toMap)
 ```
 
+### History & rollback
+
+It is possible to save the current state of RSpace in the form of a checkpoint (root hash of a given Merkle Patricia Trie).
```

Comment:
How about something like:

It is possible to save the current state of RSpace in the form of a `Checkpoint`.  A `Checkpoint` value contains the root hash of a Merkle Patricia Trie built from the contents of RSpace.